explainability-service/manifests/opendatahub/default/trustyai-deployment.yaml

```diff
@@ -40,7 +39,6 @@ metadata:
   name: trustyai-service
 spec:
   dockerImageRepository: quay.io/ruimvieira/trustyai-service
-
 ---
 apiVersion: apps.openshift.io/v1
 kind: DeploymentConfig
```

Comment:
Can you change this to kind `Deployment` and get rid of the deployment config specific things?